### PR TITLE
feat: added fleet scope logging

### PR DESCRIPTION
**Pull request by yliaog**

### CODE REVIEW

File: 4-fleetscope/envs/development/main.tf [hunk 1]
```diff
@@ -21,6 +21,11 @@ locals {
   namespace_id = "frontend"
 }
 
+import {
+  id = "projects/${var.project_id}/locations/global/features/fleetobservability"
```
Comment (apeabody) [inline 4-fleetscope/envs/development/main.tf:25]: Should be `fleet_project_id`?

```suggestion
  id = "projects/${var.fleet_project_id}/locations/global/features/fleetobservability"
```

File: 4-fleetscope/envs/non-production/main.tf [hunk 2]
```diff
@@ -21,6 +21,11 @@ locals {
   namespace_id = "frontend"
 }
 
+import {
+  id = "projects/${var.project_id}/locations/global/features/fleetobservability"
```
Comment (apeabody) [inline 4-fleetscope/envs/non-production/main.tf:25]: Should be `fleet_project_id`?

```suggestion
  id = "projects/${var.fleet_project_id}/locations/global/features/fleetobservability"
```

File: 4-fleetscope/envs/production/main.tf [hunk 3]
```diff
@@ -21,6 +21,11 @@ locals {
   namespace_id = "frontend"
 }
 
+import {
+  id = "projects/${var.project_id}/locations/global/features/fleetobservability"
```
Comment (apeabody) [inline 4-fleetscope/envs/production/main.tf:25]: Should be `fleet_project_id`?

```suggestion
  id = "projects/${var.fleet_project_id}/locations/global/features/fleetobservability"
```

File: 4-fleetscope/modules/env_baseline/log.tf [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_gke_hub_feature" "fleet-o11y" {
+  name     = "fleetobservability"
+  project  = var.project_id
```
Comment (apeabody) [inline 4-fleetscope/modules/env_baseline/log.tf:19]: Should be `fleet_project_id`?

```suggestion
  project  = var.fleet_project_id
```

**Comment by apeabody** [comment]
Hi @yliaog - Once the project_id are updated, this PR should now work.

File: 4-fleetscope/envs/development/main.tf (hunk 1 above)
Comment (yliaog) [reply 4-fleetscope/envs/development/main.tf:24]: done

File: 4-fleetscope/envs/non-production/main.tf (hunk 2 above)
Comment (yliaog) [reply 4-fleetscope/envs/non-production/main.tf:24]: done

File: 4-fleetscope/modules/env_baseline/log.tf (hunk 4 above)
Comment (yliaog) [reply 4-fleetscope/modules/env_baseline/log.tf:19]: done

File: 4-fleetscope/envs/production/main.tf (hunk 3 above)
Comment (yliaog) [reply 4-fleetscope/envs/production/main.tf:24]: done

**Comment by apeabody** [comment]
Yay!

**Review by apeabody — APPROVED** [review]
Thanks @yliaog - LGTM!

@gtsorbo?

**Review by gtsorbo — APPROVED** [review]

**Pull request merged**